src/atoms/Checkbox/CheckboxIcon.native.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon from './../Icon';
+function CheckboxIcon({ checked, size, disabled }) {
+  if (checked) {
+    return (
+      <Icon
+        size={size}
+        name="checkedSquare"
+        fill={disabled ? 'shade.300' : 'primary.800'}
+        testID="ds-checkbox-icon-checked"
+      />
+    );
+  }
+  return (
+    <Icon
+      size={size}
+      name="uncheckedSquare"
+      fill={disabled ? 'shade.300' : 'shade.500'}
+      testID="ds-checkbox-icon-unchecked"
```

Comment:
```suggestion
                {...automation(testID)}
```

Reply:
Since CheckboxIcon is used internally by <Checkbox />, why is it required to pass the testIDs to CheckboxIcon?